bot.py

```diff
@@ -52,4 +54,16 @@ async def bruh(ctx):
                              '<:dangBRUH:661420374390603776>']))
 
 
+@bot.command()
+async def schedule(ctx):
+    """Returns the most current schedule"""
+    async with aiohttp.ClientSession() as session:
+        async with session.get('https://decapi.me/twitter/latest/dangheesling?search=*Show%20Schedule%20for%20Week*') as resp:
+            respText = await resp.text()
+            dateFor = re.search('(?:\*Show Schedule for Week\* of)\s+(.\S*)', respText).group(1)
```

Comment:
The code works though I ran it against the text and a regex checker and have my doubts. I can explain how I interpret your regex. Other than that concern, I am OK with approving the PR and merging the code as the command output is successful on the bot. Then readdress the concern if the function or command breaks further down the line.

Reply:
Hey! So here is an explanation of the regex:
**dateFor**
This search is first looking for the group with the text "*Show Schedule for week* of) and not capturing this group. then looking for a space and capturing another group that is the text until the next space. This is so we can capture the date that this schedule is for. In theory, we could just capture the whole title as is but I wanted to pull the date out incase we needed it for something else.
**formatS**
This is using the sub function of the re library, this is to substitute matching groups with a replacement which I am using to break the lines of the schedule into newlines. The first part of the regex is looking for a sequence of non-digit characters (emoji + space + MON/TUE/WED etc) followed be a colon, a potential space, then 1 or 2 digits which is then followed by AM, there is an or with the repeated pattern for PM. In the substitution, I am adding a new line before these groups to be able to break them without cutting off any descriptions Dan is using (for example if there is a podcast that day)

Let me know if this makes sense or if more explanation is needed.

Thanks!